<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormManagePrevisions</class>
 <widget class="QWidget" name="FormManagePrevisions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des prévisions</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cette fenêtre permet de gérer les prévisions. Une prévision est un scénario qui calcule les valeurs des crédits alloués pour l'année à venir en utilisant des critères (fixés par l'utilisateur) sur les données historiques. Une prévision est attachée à un audit terminé, il peut y avoir plusieurs prévisions attachées à un même audit.&lt;/p&gt;&lt;p&gt;Les valeurs calculées par une prévision peuvent être utilisées pour fabriquer un audit étendu, c'est à dire un audit qui reprend les données de l'audit sur lequel est adossé la prévision et ajoute les valeur prévues calculées pour l'année à venir. Du point de vue du logiciel un audit étendu est un audit indépendant, il n'est pas lié à une prévision.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBoxPrevisions">
     <property name="title">
      <string>Gestion des prévisions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QLabel" name="labelAttachedAudit">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Créé le :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Portant sur l'arbre :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Audit associé :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelDate">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="comboListPrevisions"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Années de l'audit :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelAttachedTree">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="labelAttachedAuditYears">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="text">
         <string>&amp;Supprimer</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="pushButtonToAudit">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Créer un audit prévisionnel indépendant qui contiendra les données de l'audit existant sur lesquelles sont adossées les prévisions &lt;span style=&quot; font-weight:600;&quot;&gt;ainsi que les valeurs calculées par les critères de prévision pour l'année N+1.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Vers &amp;audit étendu</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>Nouvelle prévision</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Audit attaché </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboListAudits"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddPrevision">
        <property name="text">
         <string>Créer une &amp;nouvelle prévision</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboListPrevisions</tabstop>
  <tabstop>pushButtonDelete</tabstop>
  <tabstop>pushButtonToAudit</tabstop>
  <tabstop>comboListAudits</tabstop>
  <tabstop>pushButtonAddPrevision</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
